pkg/api/v1/types.go

```diff
@@ -2503,6 +2503,8 @@ type EventSource struct {
 	Component string `json:"component,omitempty"`
 	// Host name on which the event is generated.
 	Host string `json:"host,omitempty"`
+	// Scheduler name on which the event is generated.
+	SchedulerName string `json:"schedulerName,omitempty"`
```

Comment:
Yeah, I definitely agree with @lavalamp on all points.

Concretely, we should just make Component include the scheduler name. In principle schedulers can be very different (written in different languages, using totally different scheduling rules as long as they obey the Kubelet admission rules, etc.), so they should just be considered totally independent components. If you run multiple replicas of the same scheduler you can just number them arbitrarily, like scheduler-01, scheduler-02, etc.